make-dist.sh

```diff
@@ -30,12 +30,17 @@ else
     exit 1
 fi
 
+export MAVEN_OPTS="-Xmx2g -XX:ReservedCodeCacheSize=512m"
```

Comment:
I think you can write `export MAVEN_OPTS=${MAVEN_OPTS:-"..."}` to not overwrite the value if it's set already. Maybe you want to overwrite it every time though.